doc/source/whatsnew/v2.2.0.rst

```diff
@@ -358,6 +358,70 @@ Other API changes
 Deprecations
 ~~~~~~~~~~~~
 
+Chained assignment
+^^^^^^^^^^^^^^^^^^
+
+In preparation of larger upcoming changes to the copy / view behaviour in pandas 3.0
+(:ref:`copy_on_write`, PDEP-7), we started deprecating *chained assignment*.
+
+Chained assignment occurs when you try to update a pandas DataFrame or Series through
+two subsequent indexing operations. Depending on the type and order of those operations
+this currently does or does not work, as explained in :ref:`indexing.view_versus_copy`.
```

Comment:
To understand if your code would be affected, it can be useful how it currently works? Although, people probably only the case where it works in practice, but the other cases, well, don't work ;)